components/swap/GridListFilters.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    class="flex gap-4"
+  >
+    <ChainDropdown
+      position="bottom-auto"
+      variant="secondary-rounded"
+      :show-network-label="false"
+      :label="$t('activity.network')"
+      :exclude="['base', 'imx', 'mnt']"
+    />
```

Comment:
I don't think we should keep this since we can't mix ahp/ahk items swap (need to wait for XCM transfer)

Reply:
yeah as far as I know it's not possible atm

made it so that dropdown allows you start a swap in another chain , no mixing of items from different chains

alternatively we could just remove it